ja_jp/calendar-manage-calendars.adoc

```diff
@@ -0,0 +1,63 @@
+=== カレンダーの管理
+==== カレンダーを共有する
+アカウントで有効になっていれば、イベント、予定、ミーティングなどを含むカレンダーをほかの人と共有することができます。
+
+. btn:[カレンダー] タブを選択します。
+. 共有希望のカレンダーを右クリックし、 *共有...* をクリックします。
+. *人々をメールで招待（最も柔軟）* をクリックします。
+. *リンクを持っている人なら誰でも* のドロップダウンメニューから適切なアクセス許可を選択します。
+
+表示:: 共有先のユーザーは 非公開イベント *を除いた* すべての予定を閲覧できます。
+
+予定なし/予定ありの時間だけ表示:: 共有先のユーザーは予定の詳細を確認できず、カレンダーに予約が入っているか入っていないかのみ確認できます。
+
+表示（非公開イベントを含む）:: 共有先のユーザーは非公開のイベント *を含め* すべての予定を閲覧できます。
```

Comment:
@k-kato I would like to keep this help text as is, and change the label on UI side to "表示（非公開イベントを含む）" and "表示、編集、追加、および削除（非公開イベントを含む）" because the label "Private" in the appointment composing window is translated as "非公開".  What do you think?

Reply:
I agree. I added it to [TODO list](https://github.com/Zimbra/userguide/pull/10#issuecomment-957320690)